### PR TITLE
Fix for formula interpretation triggering in Excel

### PR DESCRIPTION
**Pull request by aeldamaty**
This fixes the problem of having text being interpreted as formulas in Excel.

### CODE REVIEW

**Comment by aeldamaty** [comment]
@funkaoshi @lmontopo there is also the change here.

**Comment by lmontopo** [comment]
This looks good. +1.

**Comment by funkaoshi** [comment]
@aeldamaty damn it my comments didn't show up because I didn't "submit my review" or something.

File: multi_import/exporter.py [hunk 1]
```diff
@@ -25,12 +25,20 @@ def export_dataset(self, template=False):
     def get_header(self):
         return self.serializer.get_fields().keys()
 
+    def escape(self, s):
```
Comment (funkaoshi) [inline multi_import/exporter.py]: Should include docstring explaining how this works.

**Comment by funkaoshi** [comment]
One comment and merge.

File: multi_import/exporter.py (hunk 1 above)
Comment (lmontopo) [inline multi_import/exporter.py]: oh yea good idea. I second that.

**Pull request merged**